packages/wallet/ui/src/bridge.jsx

```diff
@@ -126,9 +129,15 @@ const signalBridge = () => {
   }
 };
 
-ReactDOM.render(
-  <button id="signalBridge" onClick={signalBridge}>
-    Signal Bridge
-  </button>,
-  document.getElementById('root'),
-);
+const Bridge = () => {
+  return (
+    <button className="button" onClick={signalBridge}>
```

Comment:
`button.button` is a funny target. consider an `id` since there will be exactly one in the DOM:

```suggestion
    <button id="signalWallet" onClick={signalBridge}>
```

Reply:
Changed to `signal-wallet-button`

Reply:
That might have been me, aiming for type=button, which suppresses form submission, iirc